src/components/settings/SettingsLMSTab/LMSConfigs/Canvas/CanvasConfigAuthorizePage.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import isString from "lodash/isString";
+import isNumber from "lodash/isNumber";
+
+import { Form, Alert } from "@edx/paragon";
+import { Info } from "@edx/paragon/icons";
+
+// @ts-ignore
+import ValidatedFormControl from "../../../../forms/ValidatedFormControl.tsx";
+import { isValidNumber, urlValidation } from "../../../../../utils";
+import {
+  FormFieldValidation,
+  useFormContext,
+} from "../../../../forms/FormContext";
+// @ts-ignore
+import FormWaitModal from "../../../../forms/FormWaitModal.tsx";
+// @ts-ignore
+import { WAITING_FOR_ASYNC_OPERATION } from "../../../../forms/FormWorkflow.tsx";
+// @ts-ignore
+import { setWorkflowStateAction } from "../../../../forms/data/actions.ts";
+// @ts-ignore
+import { LMS_AUTHORIZATION_FAILED } from "./CanvasConfig.tsx";
+
+const formFieldNames = {
+  DISPLAY_NAME: "displayName",
+  CLIENT_ID: "clientId",
+  CLIENT_SECRET: "clientSecret",
+  ACCOUNT_ID: "canvasAccountId",
+  CANVAS_BASE_URL: "canvasBaseUrl",
+};
+
+export const validations: FormFieldValidation[] = [
+  {
+    formFieldId: formFieldNames.CANVAS_BASE_URL,
+    validator: (fields) => {
+      const error = !urlValidation(fields[formFieldNames.CANVAS_BASE_URL]);
+      return error && "Please enter a valid URL";
+    },
+  },
+  {
+    formFieldId: formFieldNames.DISPLAY_NAME,
```

Comment:
Something's going wrong and when I key smash to put in a display name, this is my error message. "canvasBaseUrlPlease enter a valid URL", so something's wrong with the variable name being appended to the validation message, but also the incorrect ones getting triggered.

Reply:
Good call out that it wasn't following the old behavior!